hw/ip/rv_core_ibex/data/rv_core_ibex.hjson

```diff
@@ -317,6 +317,36 @@
     { name: "BUS.INTEGRITY",
       desc: "End-to-end bus integrity scheme."
     }
+    { name: "CORE.SENSITIVE_DATA.SCA",
+      desc: "Data independent timing."
+    }
+    { name: "PC.CTRL_FLOW.CONSISTENCY"
+      desc: "Correct PC increment check."
+    }
+    { name: "CTRL_FLOW.UNPREDICTABLE"
+      desc: "Randomized dummy instruction insertion."
+    }
+    { name: "SENSITIVE_DATA.INTEGRITY"
+      desc: "Register file integrity checking."
+    }
+    { name: "LOGIC.CONSISTENCY"
```

Comment:
do you think it makes sense to change this to CPU_CORE.SHADOW?
it's fine to add a new category for this I think.

@msfschaffner 
@rswarbrick 
let me know if you guys approve of this suggesiton.

Reply:
Yes, that is definitely better than just LOGIC.

If we don't want to add more asset names to the scripting, we could also do something like  CPU.CTRL_FLOW.SHADOW

Reply:
I like that! Especially as the CPU isn't really an asset: we're protecting its control flow (and maybe datapath), rather than the core itself.

Reply:
I'd say we're protecting both control flow and data path. That's why I felt `LOGIC` actually fit quite well. It's broad protection for all assets that concern the CPU.

Reply:
should we go with LOGIC.SHADOW then?
In the description you can put in exactly what you're saying here Greg, that we are protecting all assets.

Reply:
I'm happy with `LOGIC.SHADOW`